manifest.xml

```diff
@@ -9,5 +9,4 @@
   <logo>http://</logo>
   <depend package="base/cmake" />
   <depend package="base/types" />
-  <depend package="drivers/hidapi" />
```

Comment:
> What is the difference of declare it here (on manifest) and declare it here?

In principle none. In practice, it's expected to be done in manifest.xml if there is one. So that's where people will look.

> It is done by adding ...

the `<rosdep ...>` tag is for backward compatibility. Just use `<depend ...>`. And yes.